### PR TITLE
Lay out ScreenOptionsService and ScreenOptionsServiceChild

### PR DESCRIPTION
**Pull request by Fieoner**
image
image

### CODE REVIEW

File: BGAnimations/ScreenOptionsServiceChild overlay.lua [hunk 1]
```diff
@@ -53,10 +57,11 @@ af[#af+1] = Def.ActorFrame{
 		InitCommand=function(self)
 			recommended_bmt = self
 
-			self:xy(- bg_width/2 + padding, bg_height/2 - padding)
-				:valign(1) -- bottom aligned
```
Comment (Sereni) [inline BGAnimations/ScreenOptionsServiceChild overlay.lua:57]: this says bottom aligned on both valign(1) and valign(0), I suspect one of those is incorrect

File: Graphics/OptionRowServiceChild Frame.lua [hunk 2]
```diff
@@ -1,21 +1,29 @@
...
+end
 }
 
 -- black quad behind the title
 t[#t+1] = Def.Quad {
 	Name="TitleBackgroundQuad",
 	OnCommand=function(self)
-		self:zoomto(title_bg_width, _screen.h*0.0625)
-			:halign(0):x(0)
+		self:setsize(title_bg_width, _screen.h*0.0625)
```
Comment (Sereni) [inline Graphics/OptionRowServiceChild Frame.lua:23]: for my education, why did you change from zoomto to setsize, and what's the difference between them?

File: metrics.ini [hunk 3]
```diff
@@ -484,16 +484,16 @@ FooterOnCommand=visible,false
...
-TitleX=WideScale(20,40)
+TitleX=36
 FrameX=0
-ItemsStartX=_screen.cx+60
-ItemsEndX=_screen.w-10
-ItemsLongRowSharedX=_screen.w/3
+ItemsStartX=_screen.cx+33
+ItemsEndX=_screen.w+48
+ItemsLongRowSharedX=_screen.w
 - 10
```
Comment (Sereni) [inline metrics.ini:491]: ^M?

Comment (Fieoner) [reply metrics.ini:491]: Windows can't newline lol

File: Graphics/OptionRowServiceChild Frame.lua (hunk 2 above)
Comment (Fieoner) [reply Graphics/OptionRowServiceChild Frame.lua:23]: They do the same I don't know why I changed it lol

**Comment by Fieoner** [comment]
second pic updated
